dotCMS/src/main/java/com/dotcms/rest/api/v1/contentImport/ContentImportForm.java

```diff
@@ -0,0 +1,63 @@
+package com.dotcms.rest.api.v1.contentImport;
+
+import com.dotcms.repackage.javax.validation.constraints.NotNull;
+import com.dotcms.rest.api.Validated;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * Form object that represents the JSON parameters for content import operations.
+ */
+public class ContentImportForm extends Validated {
+
+    @NotNull(message = "Content Type is required")
+    private final String contentType;
+
+    private final String language;
+
+    @NotNull(message = "Workflow Action ID is required")
+    private final String workflowActionId;
+
+    private final List<String> fields;
+
+    @JsonCreator
+    public ContentImportForm(
+            @JsonProperty("contentType") final String contentType,
+            @JsonProperty("language") final String language,
+            @JsonProperty("workflowActionId") final String workflowActionId,
+            @JsonProperty("fields") final List<String> fields) {
+        super();
+        this.contentType = contentType;
+        this.language = language;
+        this.workflowActionId = workflowActionId;
+        this.fields = fields;
```

Comment:
I think you have to call to    checkValid(); in order to do the actual validations